src/App.tsx

```diff
@@ -1,8 +1,17 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import cn from 'classnames';
 
-export const goodsFromServer = [
+type Goods = string [];
+
+enum SortType {
+  SORTED_BY_ALPHABET = 'alphabet',
+  SORTED_BY_LENGTH = 'length',
+  DEFAULT_FIELD = '',
+}
```

Comment:
You already said in the name of the `enum` that it will be sorting, so you can shorten the name a little:
```suggestion
enum SortType {
  BY_ALPHABET = 'alphabet',
  BY_LENGTH = 'length',
  DEFAULT_FIELD = '',
}
```